File: bddtests/src/main/kotlin/mocking/tpp/models/PatientAccess.kt
package mocking.tpp.models

import javax.xml.bind.annotation.XmlAccessType
import javax.xml.bind.annotation.XmlAccessorType
import javax.xml.bind.annotation.XmlAttribute
import javax.xml.bind.annotation.XmlElement

@XmlAccessorType(XmlAccessType.FIELD)
data class PatientAccess(
        @XmlAttribute var patientId: String = "",
        @field:XmlElement(name = "SiteDetails") var siteDetails: SiteDetails = SiteDetails()
)

File: xamarinintegrationtests/NHSOnline.HttpMocks/Tpp/Models/PatientAccess.cs
using System.Xml.Serialization;

namespace NHSOnline.HttpMocks.Tpp.Models
{
    public sealed class PatientAccess
    {
        [XmlAttribute("patientId")]
        public string? PatientId { get; set; }

        public SiteDetails? SiteDetails { get; set; }
    }
}